make/deps.mk

```diff
@@ -53,8 +53,13 @@ ifeq ($(CIRCLECI),true)
 endif
 
 dep_ranch = git https://github.com/2600hz/erlang-ranch 1.7.1
-dep_cowboy = git https://github.com/2600hz/erlang-cowboy 2.8.0-OTP19
-dep_cowlib = git https://github.com/2600hz/erlang-cowlib 2600hz-2.9.1-4.3
+
+#dep_cowboy = git https://github.com/2600hz/erlang-cowboy 2.8.0-OTP19
+#dep_cowlib = git https://github.com/2600hz/erlang-cowlib 2600hz-2.9.1-4.3
```

Comment:
Don't need extra commented out dependencies left in commit, if you can clean this up I can merge.